src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Tables/DeltaTable.cs

```diff
@@ -19,6 +20,7 @@ namespace Microsoft.Spark.Extensions.Delta.Tables
     /// </summary>
     public class DeltaTable : IJvmObjectReferenceProvider
     {
+        private const string JvmClassName = "io.delta.tables.DeltaTable";
```

Comment:
Let's use `private static readonly string s_deltaTableClassName` similar to how we're doing it for other classes, ie `Functions.cs`, `SparkSession.cs`

Reply:
Why static and not const?

Reply:
Either should be fine since it's `private`.  We should just make sure we're consistent with our declaration.  So either we can keep this as `const`, and change the other classes, or change this to `static readonly`

Reply:
Thoughts @imback82 ?

Reply:
Either one is fine. So in this case, I will choose consistency.

Reply:
Ok, changed to static.